go/vt/sqlparser/sql.y

```diff
@@ -319,6 +320,12 @@ select_statement:
     $$ = &Select{Comments: Comments($2), Cache: $3, SelectExprs: SelectExprs{Nextval{Expr: $5}}, From: TableExprs{&AliasedTableExpr{Expr: $7}}}
   }
 
+stream_statement:
+  STREAM comment_opt select_expression FROM table_name
+  {
+    $$ = &Stream{Comments: Comments($2), SelectExpr: $3, Table: $4}
```

Comment:
'Table: $4' is wrong, should be 'Table: $5'

Reply:
Thanks, that fixed it immediately!